Extensions/editable_reblogs.js

```diff
@@ -13,7 +13,7 @@ XKit.extensions.editable_reblogs = new Object({
 		this.running = true;
 
 		XKit.interface.post_window_listener.add("editable_reblogs", XKit.extensions.editable_reblogs.post_window);
-		XKit.tools.add_css(".control-reblog-tree {display: none; }", "editable_reblogs_remove_content_tree");
+		XKit.tools.add_css(".post-form--header .reblog-title {margin: 10px;} .", "editable_reblogs_remove_content_tree");
```

Comment:
Hmm. I did at the time, because that class isn't used in the new design -- but now I realize I should have left it for the fallback. Will put it back when I get home. :)